### PR TITLE
bootstrap4 theme for dropdowns

### PR DESCRIPTION
**Pull request by orthagh**
Before:
image

After: 
image

### CODE REVIEW

**Comment by orthagh** [comment]
added to the pr some fast tweaks to the main view:

After: 
image

Before:
image

**Comment by orthagh** [comment]
last iteration: 
Image collée à 2019-3-22 11-48

File: templates/edit_page.twig [hunk 1]
```diff
@@ -41,35 +41,31 @@
...
-      {% endif %}
-                     </a>
-                  </li>
-   {% endfor %}
-               </ul>
-            </div>
-         </div>
-      </div>
-      <div class="col-md-9 tab-content">
+      <!-- <div class="col-md-3"> -->
```
Comment (trasher) [inline templates/edit_page.twig]: maybe drop the comment?

File: templates/edit_page.twig [hunk 2]
```diff
@@ -41,35 +41,31 @@
...
+{% if entry.count is defined %}
+                  <span class="pull-right-container">
+                     <span class="badge badge-{% if entry.count == 0 %}light{% else %}info{% endif %} float-right">{{ entry.count }}</span>
+                  </span>
+{% endif %}
+               </a>
+            </li>
+{% endfor %}
+         </ul>
+      <!-- </div> -->
```
Comment (trasher) [inline templates/edit_page.twig]: Same

**Pull request merged**